README.md

```diff
@@ -237,388 +104,96 @@ You need to copy the following files:
-described in section 4.2. If you do not have an openttd.cfg OpenTTD will
-create one after closing.
-
-## 7.0) Translating
-
-See https://www.openttd.org/development for up-to-date information.
-
-The use of the online Translator service, located at
-https://translator.openttd.org, is highly encouraged. For getting an account
-simply follow the guidelines in the FAQ of the translator website.
-
-If for some reason the website is down for a longer period of time, the
-information below might be of help.
-
-Please contact the translations manager (https://www.openttd.org/contact)
-before beginning the translation process! This avoids double work, as
-someone else may have already started translating to the same language.
-
-### 7.1) Translation
-
-So, now that you have notified the development team about your intention to
-translate (You did, right? Of course you did.) you can pick up english.txt
-(found in the Git repository under /src/lang) and translate.
-
-You must change the first two lines of the file appropriately:
-
-    ##name English-Name-Of-Language
-    ##ownname Native-Name-Of-Language
-
-Note: Do not alter the following parts of the file:
-
-- String identifiers (the first word on each line)
-- Parts of the strings which are in curly braces (such as {STRING})
-- Lines beginning with ## (such as ##id), other than the first two lines
-   of the file
-
-### 7.2) Previewing
-
-In order to view the translation in the game, you need to compile your language
-file with the strgen utility. As this utility is tailored to a specific OpenTTD
-version, you need to compile it yourself. Just take the normal OpenTTD sources
-and build that. During the build process the strgen utility will be made.
-
-strgen is a command-line utility. It takes the language filename as parameter.
-
-Example:
-
-    strgen lang/german.txt
-
-This results in compiling german.txt and produces another file named german.lng.
-Any missing strings are replaced with the English strings. Note that it looks
-for english.txt in the lang subdirectory, which is where your language file
-should also be.
-
-That is all! You should now be able to select the language in the game options.
-
-## 8.0) Troubleshooting
-
-To see all startup options available to you, start OpenTTD with the
-'`./openttd -h`' option. This might help you tweak some of the settings.
-
-If the game is acting strange and you feel adventurous you can try the
-'`-d [[<name>=]<level>]`' flag, where the higher levels will give you more
-debugging output. The 'name' variable can help you to display only some type of
-debugging messages. This is mostly undocumented so best is to look in the
-source code file debug.c for the various debugging types. For more information
-look at https://wiki.openttd.org/Command_line.
 
-The most frequent problem is missing data files. Please install OpenGFX and
-possibly OpenSFX and OpenMSX. See section 4.1.1 for more information.
+## 1.5) Add-on content / mods
```

Comment:
some introductory paragraph in here along the lines of:
"Various methods to change the game content are available. Most of them can be installed with the ingame downloader. If you want to install them manually, refer to [it's complicated]"

Reply:
Should we also reference patchpacks here?

Reply:
Patchpacks are unofficial versions, so I think we are not supposed to mention them

Reply:
Patchpacks are mentioned in the official goals https://github.com/OpenTTD/OpenTTD/blob/master/CONTRIBUTING.md#i-do-not-agree-with-the-goals-of-the-official-branch-what-can-i-do-instead

BUT I think there's a difference between two messages:
* saying to developers 'we support forks and patchpacks" is fine
* saying to players "if you want to try different gameplay, download random unknown modified OpenTTD binaries" is not fine

Reply:
I think leaving out patchpacks here is fine.